mon-pix/tests/unit/services/oidc-identity-providers-test.js

```diff
@@ -1,7 +1,7 @@
 import Object from '@ember/object';
 import Service from '@ember/service';
 import { setupTest } from 'ember-qunit';
-import { assert, module, test } from 'qunit';
+import { assert, context, module, test } from 'qunit';
```

Comment:
Il faut enlever `context` ici aussi : 
```suggestion
import { assert, module, test } from 'qunit';
```